packages/cozy-konnector-libs/docs/api.md

```diff
@@ -905,6 +906,38 @@ Get the data saved by saveAccountData
 Update account attributes and cache the account
 
 **Kind**: instance method of [<code>BaseKonnector</code>](#BaseKonnector)  
+<a name="BaseKonnector+waitForTwoFaCode"></a>
+
+### baseKonnector.waitForTwoFaCode()
+Tells Home that 2FA code is needed and wait for the user to submit it.
```

Comment:
I would say, 'notice that` or `tell Harvest` instead of `tell Home that` since it's not only about the Home app here

Reply:
I pick `notice that` because connector contributors probably won't know what harvest is I suppose.